app/src/main/res/values/strings.xml

```diff
@@ -815,6 +817,8 @@
     <string name="about_your_rights">Your rights</string>
     <!-- About page link text to open open source licenses screen -->
     <string name="about_open_source_licenses">Open source libraries we use</string>
+    <!-- About page link text to open what's new link -->
+    <string name="about_whats_new">What\'s new in %s</string>
```

Comment:
Is this what TC doesn't like? Can you have an apostrophe?

Reply:
I'm just getting a 404 error 😕  I can try that though!